app/assets/v2/js/user-search.js

```diff
@@ -1,7 +1,7 @@
 $(function() {
   $('.username-search').select2({
     ajax: {
-      url: '/api/v0.1/users_search/',
+      url: '/api/v0.1/users_search/?token=' + currentProfile.githubToken,
```

Comment:
Should we put in a check there to ensure the token is available ?

Reply:
fixed it here https://github.com/gitcoinco/web/pull/2296

Reply:
Yeah 😛